apiaudio/api_resources/voiceCloner.py

```diff
@@ -0,0 +1,136 @@
+from typing import Literal, Optional, List, Union
+
+from apiaudio.api_request import APIRequest
+
+
+class UsersManager(APIRequest):
+    """
+    CRUD interface for Users resource.
+
+    ...
+
+    Notes:
+    - A user should represent a single person/voice
+    - Verticals are assigned to a user
+    - Uploaded files belong to a user
+    - Trained models belong to a user
+
+    """
+
+    OBJECT_NAME = "VC_Users"
```

Comment:
This is different to the rest of the naming conventions in other objects

Reply:
Yeah, didn't want to name it users as it's too broad and might appear somewhere else